### DISCUSSION TITLE
on hover on the bar of the bar chart the mouse doesn't changes to pointer

### DISCUSSION THREAD

**Discussion by abhay-archive** [plouc/nivo#2150] [Q&A]
on hover on the bar of the bar chart the mouse doesn't changes to pointer

**Comment by amancognisaas** [comment]
To Change cursor to pointer in nivo charts
This Worked for me, Try this,
```
onMouseEnter={(event) => {
    (document.getElementById('chartsMouse') as HTMLInputElement).style.cursor = 'pointer';
}}
onMouseLeave={(event) => {
    (document.getElementById('chartsMouse') as HTMLInputElement).style.cursor = 'default';
}}
```
since there is no way to access cursor in nivo svg here we targeted parent element and make its cursor to pointer and on mouse leave again make it normal

**Comment by abhay-archive** [comment]
@amancognisaas Thanks a lot. I added the id to the parent element and this worked.
